tests/app/parser/test_metadata_parser.py

```diff
@@ -144,6 +144,35 @@ def test_deserialisation_iso_8601_dates(fake_metadata_runner):
     assert isinstance(claims["birthday"], str)
 
 
+def test_deserialisation_iso_8601_datetime(fake_metadata_runner):
+    claims = validate_runner_claims(fake_metadata_runner)
+
+    assert claims["response_expires_at"] == "2021-11-22T15:34:54+00:00"
+
+
+def test_deserialisation_iso_8601_datetime_with_zulu(fake_metadata_runner):
+    claims = validate_runner_claims(fake_metadata_runner)
+    fake_metadata_runner["response_expires_at"] = "1900-11-22T15:34:54Z"
+
+    assert claims["response_expires_at"] == "2021-11-22T15:34:54+00:00"
```

Comment:
This doesn't make sense to me. It needs to assigned before you validate, and also need to be a date in the future?

Reply:
Yep I mucked this one up, should be sorted now.